packages/editor/src/components/editor-regions/index.js

```diff
@@ -18,10 +18,16 @@ function EditorRegions( {
 	className,
 } ) {
 	return (
-		<div className={ classnames( className, 'edit-post-editor-regions' ) }>
+		<div
+			className={ classnames(
+				className,
+				'components-editor-regions',
```

Comment:
The convention is not right, it should be `editor-editor-regions` (same for the sub classes).
Also wondering if there's a better name than EditorRegions

Reply:
> The convention is not right, it should be `editor-editor-regions` (same for the sub classes).

Yeah, was wondering about the class prefix. FWIW, I went followed precedent in `packages/editor/src/components/`: https://github.com/WordPress/gutenberg/blob/164d5830a9acd895dfd661b5fde1ca8b80b270ac/packages/editor/src/components/editor-notices/index.js#L35

> Also wondering if there's a better name than EditorRegions

:thinking: Any suggestions? Maybe... EditorScaffolding? EditorSkeleton? EditorChrome?

Reply:
I think I like both EditorSkeleton and EditorChrome. Maybe more Skeleton? thoughts?
And yeah that notices class name is wrong too.

Reply:
Yeah, Chrome could be misleading. I hope 'Skeleton' won't be off-putting for people? :sweat_smile: :skull:  EditorStructure maybe?

Reply:
I went with `EditorSkeleton` :slightly_smiling_face: